llvm/lib/Target/LoongArch/LoongArchISelLowering.h

```diff
@@ -301,7 +301,8 @@ class LoongArchTargetLowering : public TargetLowering {
   SDValue getAddr(NodeTy *N, SelectionDAG &DAG, CodeModel::Model M,
                   bool IsLocal = true) const;
   SDValue getStaticTLSAddr(GlobalAddressSDNode *N, SelectionDAG &DAG,
-                           unsigned Opc, bool UseGOT, bool Large = false) const;
+                           unsigned Opc, bool UseGOT, bool Large = false,
+                           bool IsNormalOrMediumLE = false) const;
```

Comment:
This parameter should be removable.
`IsNormalOrMediumLE` = `(Opc == LoongArch::PseudoLA_TLS_LE && !Large)`

Reply:
When TLS_LE calling `getStaticTLSAddr()`, parameter `Large` always uses the default value `false` no matter what the actual codemodel is. So `LoongArch::PseudoLA_TLS_LE && !Large` is always `true`. But we only expect `IsNormalOrMediumLE` to be true when the codemodel is actually normal or medium.

Reply:
Sorry, I didn’t notice that. In that case, can we actually pass Large here and handle different Opc values accordingly?

Reply:
I have modified according to your suggestions. Thanks.